eslint.config.js

```diff
@@ -292,11 +292,11 @@ module.exports = [
     })),
     {
         name: "eslint/ts-rules",
-        files: ["tests/lib/types/*.ts"],
+        files: ["tests/lib/types/*.ts", "packages/**/**/*.{ts,mts,cts,tsx}"],
```

Comment:
```suggestion
        files: ["tests/lib/types/*.ts", "packages/**/*.{ts,mts,cts,tsx}"],
```

Consecutive `/**` seem unnecessary?